lib/shovel.js

```diff
@@ -160,7 +168,8 @@ function downloadFromGithub(opts, resolve) {
       bash: `
                 cd ${opts.dir}
                 wget -qO- ${repo} | tar xvz --strip-components=1
-            `
+            `,
+      dir: opts.dir,
```

Comment:
`util.codeWriteSync` worked without this because it defaults to `/home/codewarrior` when directory is falsy (`undefined`).